app/src/androidTest/java/com/duckduckgo/app/browser/animations/TrackersRendererTest.kt

```diff
@@ -14,20 +14,16 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.privacy.dashboard.impl.animations
+package com.duckduckgo.app.browser.omnibar.animations
```

Comment:
As above, the package or the location of the file is wrong.

`package com.duckduckgo.app.browser.omnibar.animations` but the file is in `com.duckduckgo.app.browser.animations`